.github/workflows/run-tests.yml

```diff
@@ -79,54 +79,54 @@ jobs:
         working-directory: ./kpwn_db
         run: ./test.sh
 
-      - name: Build expkit
+      - name: Build libxdk
         if: ${{ success() || failure() }}
-        working-directory: ./expkit
+        working-directory: ./libxdk
         run: ./build.sh
 
-      - name: Test expkit (local tests only)
+      - name: Test libxdk (local tests only)
         if: ${{ success() || failure() }}
-        working-directory: ./expkit
+        working-directory: ./libxdk
         run: ./run_local_tests.sh
 
-      - name: Test expkit (QEMU)
+      - name: Test libxdk (QEMU)
         if: ${{ success() || failure() }}
-        working-directory: ./expkit
-        run: CUSTOM_MODULES_KEEP=1 timeout 2m ./run_tests.sh ${{ steps.vars.outputs.target }} 20 --tap
```

Comment:
What was the issue with `CUSTOM_MODULES_KEEP=1`? In theory there could be two issues:
* it tries to recompile the kernel module which makes test slower
* there is a possibility of race if tests run in parallel, that multiple threads write the same file (as part of the kernel module compilation), leading to corrupt kernel module